src/common/i18n/translations/zh.json

```diff
@@ -0,0 +1,363 @@
+  "idScanner": {
+    "enterIdManually": "输入证件号码",
+    "enterManually": "自己输入",
+    "enterVoucherId": "输入优惠券编号",
+    "scanToCheck": "扫描查看有效"
+  },
+  "navigationDrawer": {
+    "logout": "登出",
+    "addCampaign": "添加活动",
+    "selectCampaign": "选活动",
+    "changeChampaign": "更改活动",
+    "helpSupport": "支援服务",
+    "termsOfUse": "使用条款",
+    "privacyStatement": "隐私声明",
+    "reportVulnerability": "报告漏洞"
+  },
+  "customerQuotaScreen": {
+    "campaignExpiredOn": "在 %{dateTime} 已到期",
+    "campaignValidTo": "有效期至 %{dateTime}",
+    "quotaRedeemedLimitReachedIDNumber": "证件号码",
+    "quotaCategoryMealCredits": "餐食积点",
+    "quotaLimitMax": "最多",
+    "quotaAppealCancel": "取消",
+    "quotaButtonCheckout": "确认交易",
+    "quotaButtonAdd": "+ 添加",
+    "quotaButtonAddVoucher": "添加优惠券",
+    "quotaAddId": "输入另一个证件号码已增加限额",
+    "quotaCategoryCommunityFoodPack": "社区粮食礼包",
+    "quotaCategoryBread": "面包",
+    "quotaCategoryEggs": "鸡蛋",
+    "quotaCategoryFruits": "水果",
+    "quotaCategoryRootVegetables": "根茎类蔬菜",
+    "quotaCategoryFreshVegetables": "新鲜蔬菜",
+    "quotaCategoryChicken": "鸡肉（新鲜/冷冻/加工）",
+    "quotaCategoryCannedFood": "罐头食品（肉类/海鲜/蔬菜）",
+    "quotaCategoryPaperProducts": "纸制品",
+    "quotaCategoryInstantNoodles": "快熟面",
+    "quotaCategoryRice": "米",
+    "quotaCategoryCookingOil": "食用油",
+    "quotaCategoryTTToken": "TT 防疫器",
+    "quotaCategoryVouchers": "优惠券",
+    "quotaCategoryAppreciationGift": "礼包",
+    "quotaIdentifierDeviceCode": "配备编号",
+    "quotaIdentifierContactNumber": "联络号码",
+    "quotaIdentifierFirstVoucherCode": "首张优惠券编号",
+    "quotaIdentifierButtonScan": "扫描",
+    "quotaScanButtonBack": "返回",
+    "merchantCode": "商人编号"
+  },
+  "checkoutSuccessScreen": {
+    "redeemed": "已领取!",
+    "redeemedItems": "已领取物品：",
+    "redeemedNextIdentity": "下一个证件",
+    "purchased": "已购买!",
+    "purchasedItems": "已购买物品：",
+    "purchasedNextIdentity": "下一个证件",
+    "previouslyRedeemedItems": "已领取物品：",
+    "previouslyPurchasedItems": "已购买物品：",
+    "redeemedAgo": "在 %{time} 已领取",
+    "redeemedOn": "在 %{time} 已领取",
+    "previouslyRedeemed": "已领取过",
+    "showMore": "显示更多内容",
+    "showLess": "隐藏内容",
+    "today": " 今天。",
+    "limitReachedRecent": "%{time} 前达到领取限额",
+    "limitReachedDate": "在 %{dateTime} 达到领取限额",
+    "limitReached": "达到领取限额",
+    "redeemedLimitReached": "到 %{date} 为止还有%{quantity}件物品",
+    "valid": "有效",
+    "complete": "确认交易"
+  },
+  "customerAppealScreen": {
+    "raiseAppeal": "提出上诉",
+    "indicateReason": "说明上诉理由",
+    "lost/stolen": "代币遗失/被盗",
+    "deadBattery": "电池没电",
+    "damagedToken": "代币损坏",
+    "oldDeviceCode": "旧配备编号",
+    "newDeviceCode": "新配备编号"
+  },
+  "statisticsScreen": {
+    "distributedAmount": "您已分发%{amount}",
+    "lastDistributedTiming": "最后在 %{dateTime} 分发",
+    "viaAppeal": "通过上诉"
+  },
+  "notEligibleScreen": {
+    "notEligible": "不符合资格",
+    "logAppeal": "请提出上诉。"
+  },
+
+  "thrownErrors": {
+    "campaignConfigNotLoad": "活动配置未加载",
+    "updateCheckError": "尝试更新活动格局时出错:",
```

Comment:
Is the colon at the end intended?

Reply:
Yep used for this: 
```
const error = new UpdateError(
      `${i18n.t("thrownErrors.updateCheckError")} ${lastUpdateResult}`
    );
```

Reply:
Maybe can leave the colon in the component itself

```
const error = new UpdateError(
      `${i18n.t("thrownErrors.updateCheckError")}: ${lastUpdateResult}`
    );
```

so it is clearer what is the "sentence structure" of the displayed copy. But this is just a small nit, ok with either